WordPress/Classes/Networking/Plugin Management/SelfHostedPluginManagementClient.swift

```diff
@@ -0,0 +1,169 @@
+
+class SelfHostedPluginManagementClient: PluginManagementClient {
```

Comment:
This isn't in WPKit because I need the reference to `JetpackSiteRef`. If not for that this would live there.